netci.groovy

```diff
@@ -71,7 +71,7 @@ set TMP=%TEMP%
 
       def triggerPhraseOnly = false
       def triggerPhraseExtra = ""
-      Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
+      Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-vs15-0')
```

Comment:
I would have assumed this was a VS2015 image. dev15.0 would probably be a better name.